content/hardware/04.pro/boards/portenta-h7/tutorials/lauterbach-debugger/content.md

```diff
@@ -1,83 +1,95 @@
 
-Enter the board's serial number (instructions below), your name and e-mail address and you will be sent your license key.
+- In the Arduino IDE, select the "Tools->Get Board Info" menu command after choosing the port to which your board is connected. This should show a 16-character (64-bit) long serial number for your Nano BLE devices and a 24-character (96-bit) long serial number for your Portenta H7 or Nicla Vision.
 
-![Request a Debug License for Arduino Pro](assets/por_ard_trace32_register.png)
+![Your board's serial number can be displayed in the Arduino IDE using the "Get Board Info" command](assets/por_ard_trace32_board_info.png)
 
-There are two alternative ways to detect the board serial number:
+***For Portenta H7 and Nicla Vision only: if you only see a 16-character (64-bit) long serial number, then you need to update your Arduino IDE and your boards' core from the Boards Manager in the IDE. Details on how to do so can be found on the Arduino website. In addition, make sure your board has the [latest bootloader](https://docs.arduino.cc/tutorials/portenta-h7/updating-the-bootloader) installed.***
 
-- In the Arduino IDE select the "Tools->Get Board Info" menu command after selecting the port to which the Portenta is connected to. This should show a 24 character (96 bit) long serial number.
+- Use the TRACE32 debugger. Check the section "Starting the TRACE32 Debugger" to learn how to get started. Click the menu item "Board S/N and License State". Your board's serial number will be printed in the AREA window and a dialog LICENSE.state will be opened.
 
-![Portenta's serial number can be displayed in the Arduino IDE using the "Get Board Info" command](assets/por_ard_trace32_board_info.png)
+![Click the menu item "Detect Board S/N" to display the serial number](assets/por_ard_trace32_board_sn.png)
 
-***If you only see a 16 character (64-bit) long serial number, then you need to update your Arduino IDE and the "Arduino mbed-enabled Boards" core from the boards manager in the IDE. Details on how to do this can be found on the Arduino website. Also, make sure your Portenta H7 has the [latest bootloader](https://docs.arduino.cc/tutorials/portenta-h7/updating-the-bootloader) installed.***
+![License State Dialog](assets/por_ard_trace32_license_state.png)
 
-- Use the TRACE32 debugger. Check section "3. Start the TRACE32 Debugger" on how to start it. Click the menu item "Board S/N and License State". Your board's serial number will be printed in the AREA window and a dialog LICENSE.state will be opened.
+***Note: Newer TRACE32 software version should automatically detect and show the board serial number, at the first connection to the target.***
 
-![Click the menu item "Detect Board S/N" to display the serial number](assets/por_ard_trace32_board_sn.png)
+At this point, you are ready to register at the Lauterbach registration page below:
```

Comment:
The preview is not good so 
![image](https://github.com/arduino/docs-content/assets/7687624/3255f2a2-366c-4126-bf74-37eca9b19a9d)

better to do something like this to avoid format errors and artifacts:

```suggestion
At this point, you are ready to register at the [Lauterbach registration page](http://www.lauterbach.com/4543):
```

Reply:
Right perfect, thanks!